### DISCUSSION TITLE
MIMIC Services

### DISCUSSION THREAD

**Discussion by syf720** [MIT-LCP/mimic-code#1669] [MIMIC-IV] [answered]
Hello! I am a student looking at patient flow for my senior thesis. I am looking at the services table and would like to understand the **services patients are in beyond their abbreviations**. I have made the following guesses, but was hoping that someone could **confirm and/or correct my guess** as I wasn't able to find further information about this in the documentation. In particular, I was unsure about what MED, NMED, and OMED mapped to. I have excluded PSYCH and GYN from my analysis. This is what I have right now:

MED | General/Internal Medicine
CMED | Cardiology
NMED | Neurology
OMED | Oncology
SURG | (General) Surgery
CSURG | Cardiac Surgery
NSURG | Neural Surgery
ORTHO | Orthopedics
TRAUM | Trauma
TSURG | Thoracic Surgery
VSURG | Vascular Surgery

Thank you so much for your help!

**Comment by alistairewj** [comment, marked as answer]
They are listed out here: https://mimic.mit.edu/docs/iv/modules/hosp/services/

From that page:

Service | Description
--- | ---
CMED | Cardiac Medical - for non-surgical cardiac related admissions
CSURG | Cardiac Surgery - for surgical cardiac admissions
DENT | Dental - for dental/jaw related admissions
ENT | Ear, nose, and throat - conditions primarily affecting these areas
EYE | Eye diseases - including subspecialty services in glaucoma, cataract surgery, cornea and external diseases, and neuro-ophthalmology.
GU | Genitourinary - reproductive organs/urinary system
GYN | Gynecological - female reproductive systems and breasts
MED | Medical - general service for internal medicine
NB | Newborn - infants born at the hospital
NBB | Newborn baby - infants born at the hospital 
NMED | Neurologic Medical - non-surgical, relating to the brain
NSURG | Neurologic Surgical - surgical, relating to the brain
OBS | Obstetrics - conerned with childbirth and the care of women giving birth
ORTHO | Orthopaedic - surgical, relating to the musculoskeletal system
OMED | Orthopaedic medicine - non-surgical, relating to musculoskeletal system
PSURG | Plastic - restortation/reconstruction of the human body (including cosmetic or aesthetic)
PSYCH | Psychiatric - mental disorders relating to mood, behaviour, cognition, or perceptions
SURG | Surgical - general surgical service not classified elsewhere
TRAUM | Trauma - injury or damage caused by physical harm from an external source
TSURG | Thoracic Surgical - surgery on the thorax, located between the neck and the abdomen
VSURG | Vascular Surgical - surgery relating to the circulatory system

**Reply by syf720** [reply]
Thank you--That page is very helpful!
